src/refmvs.rs

```diff
@@ -36,12 +36,14 @@ pub struct RefMvsTemporalBlock {
     pub mv: Mv,
     pub r#ref: i8,
 }
+const _: () = assert!(mem::size_of::<RefMvsTemporalBlock>() == 5);
 
 #[derive(Clone, Copy, PartialEq, Eq, FromZeroes)]
 #[repr(C)]
 pub struct RefMvsRefPair {
     pub r#ref: [i8; 2],
 }
+const _: () = assert!(mem::size_of::<RefMvsRefPair>() == 2);
```

Comment:
Can you add a similar comment to the one on `RefMvsTemporalBlock` that this is packed, as verified by the size check, but we can't do both packed and aligned in Rust?